datahub-web-react/src/app/entity/shared/containers/profile/header/EntityHeader.tsx

```diff
@@ -66,6 +66,8 @@ export function getCanEditName(
             return privileges?.manageGlossaries || !!entityData?.privileges?.canManageEntity;
         case EntityType.Domain:
             return privileges?.manageDomains;
+        case EntityType.DataProduct:
+            return true; // TODO: add permissions for data products
```

Comment:
Followup?

Reply:
yeah same here I think this is a good followup piece with the one above since we do handle the check on the backend and throw an understandable error if they try to edit without being able to do so